src/tir/transforms/inject_ptx_async_copy.cc

```diff
@@ -113,9 +116,11 @@ class PTXAsyncCopyInjector : public StmtMutator {
             return PrimExpr();
           }();
           if (src_offset.defined() && dst_offset.defined()) {
```

Comment:
Could you add one unittest for related circumstance? thank you!

Reply:
[here](https://gist.github.com/JackWeiw/5b80956ab44c0f63d4f434f18f42cc89) is the script to reproduce dtype mismatch in InjectPTXAsyncCopy PASS.
Sorry i missed this suggestion in ealier days. It could be a little hard to add one unittest for this related circumstance.
I can't give a conclusion for this related circumstance. Could u give some advice on how to add unittest for this related circumstance?